src/execution-metrics.ts

```diff
@@ -14,6 +14,7 @@ export class ExecutionMetrics {
     public skippedSpecs = 0;
     public totalSpecsDefined = 0;
     public executedSpecs = 0;
+    public errors: CustomReporterResult[] = [];
```

Comment:
I was planning to cheat and use the same mechanism in karma-jasmine, for global errors.

In karma-jasmine, global errors have to be passed from the browser back to the server, then reported via the karma-reporter. Our reporter translates karma API into jasmine calls then into jasmine-spec-reporter.  The `jasmineDone()` api provides a path to report global errors.

That case need not concern you, so my only other question is whether there could be other errors in future? The `jasmineDone()`/`suiteDone()` reporting path is not specific to `afterAll()`, but then again I don't know of any other possible errors.

Not a big deal, so decide and I will follow up.